### PR TITLE
Add MACCADDR8 for PGCompiler

### PR DESCRIPTION
**Pull request by asimfarooq5**
### Description
Add MACCADDR8 for PGCompiler

### CODE REVIEW

File: lib/sqlalchemy/dialects/postgresql/base.py [hunk 1]
```diff
@@ -1517,6 +1517,15 @@ def update():
     UUID: PGUuid,
 }
 
+
+class MACADDR8(MACADDR):
+    __visit_name__ = "MACADDR8"
+
+
+class TIMESTAMPTZ(TIMESTAMP):
```
Comment (zzzeek) [inline lib/sqlalchemy/dialects/postgresql/base.py:1525]: TIMESTAMPTZ is just a PG-specific abbreviation for TIMESTAMP WITH TIMEZONE and is unnecessary, so remove this

**Review by zzzeek — CHANGES_REQUESTED** [review]
hi and thanks for this!

Please remove TIMESTAMPTZ as it is not needed, and please add a MACADDR8 column (with a distinct name) to:

https://github.com/sqlalchemy/sqlalchemy/blob/a134ec1760df6295d537ff63df7aee83d957bf6a/test/dialect/postgresql/test_types.py#L2955

thanks!

File: lib/sqlalchemy/dialects/postgresql/base.py (hunk 1 above)
Comment (asimfarooq5) [reply lib/sqlalchemy/dialects/postgresql/base.py]: Please, review the changes.

File: lib/sqlalchemy/dialects/postgresql/base.py [hunk 2]
```diff
@@ -1517,6 +1517,11 @@ def update():
     UUID: PGUuid,
 }
 
+
+class MACADDR8(MACADDR):
```
Comment (CaselIT) [inline lib/sqlalchemy/dialects/postgresql/base.py:1521]: please move this to the same file where macaddr is defined

**Review by CaselIT — CHANGES_REQUESTED** [review]

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 837a68eba3e31e0acbb7c47ee87bca4e9def7648 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 837a68eba3e31e0acbb7c47ee87bca4e9def7648: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4038

**Comment by CaselIT** [comment]
Sorry, this was forgotten. I'll try taking a better look this week

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 837a68eba3e31e0acbb7c47ee87bca4e9def7648 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Patchset 837a68eba3e31e0acbb7c47ee87bca4e9def7648 added to existing Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4038

**Comment by sqla-tester** [comment]
**Federico Caselli** (CaselIT) wrote:

Yes, I'll try to bring this to the finishing line

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4038

**Comment by zzzeek** [comment]
do we have an issue number and proposed milestone for this?

**Comment by CaselIT** [comment]
There is no issue number.
I think v2 is fine

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4038 has been **merged**. Congratulations! :)

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4318 has been **merged**. Congratulations! :)

**Pull request closed without merging**